docs/sources/developers/plugins/build-a-streaming-data-source-plugin.md

```diff
@@ -4,29 +4,35 @@ title: Build a streaming data source plugin
 
 # Build a streaming data source plugin
 
-This guide explains how to build a streaming data source plugin.
+In Grafana, you can set your dashboards to automatically refresh at a certain interval, no matter what data source you use. Unfortunately, this means that your queries are requesting all the data to be sent again, regardless of whether the data has actually changed. The solution to this issue is to add _streaming_ to the plugin.
```

Comment:
I would extend on why it is the solution here rather than below. My suggestion would be "to update your dashboard only when new data becomes available."

Reply:
Fixed in latest commit.